### PR TITLE
[Workflow] Do to talk about workflow, and explain what support option is for

### PR DESCRIPTION
**Pull request by lyrixx**
Again, registry is root of so many confusion, we should not talk about it at all

### CODE REVIEW

File: workflow/workflow-and-state-machine.rst [hunk 1]
```diff
@@ -81,6 +81,7 @@ Below is the configuration for the pull request state machine.
                     marking_store:
                          type: 'method'
                          property: 'currentPlace'
+                    # The supports options is useful only if you are using twig functions ('workflow_*')
```
Comment (xabbuh) [inline workflow/workflow-and-state-machine.rst:84]:
```suggestion
                    # the "supports" option is useful only if you are using Twig functions ('workflow_*')
```

File: workflow/workflow-and-state-machine.rst [hunk 2]
```diff
@@ -132,6 +133,7 @@ Below is the configuration for the pull request state machine.
                         <framework:property>currentPlace</framework:property>
                     </framework:marking-store>
 
+                    <!-- The supports options is useful only if you are using twig functions ('workflow_*') -->
```
Comment (xabbuh) [inline workflow/workflow-and-state-machine.rst:136]:
```suggestion
                    <!-- The "support" option is useful only if you are using Twig functions ('workflow_*') -->
```

File: workflow/workflow-and-state-machine.rst [hunk 3]
```diff
@@ -202,6 +204,7 @@ Below is the configuration for the pull request state machine.
 
             $pullRequest
                 ->type('state_machine')
+                // The supports options is useful only if you are using twig functions ('workflow_*')
```
Comment (xabbuh) [inline workflow/workflow-and-state-machine.rst:207]:
```suggestion
                // The "supports" option is useful only if you are using Twig functions ('workflow_*')
```

**Comment by javiereguiluz** [comment]
Thanks Grég!

File: workflow/workflow-and-state-machine.rst (hunk 1 above)
Comment (javiereguiluz) [reply workflow/workflow-and-state-machine.rst:84]: I did these changes while merging. Thanks!

**Pull request merged**